src/main/java/com/intrafind/log4shell/Log4shellUpdate.java

```diff
@@ -37,11 +47,26 @@
 @SuppressWarnings("ArraysAsListWithZeroOrOneArgument")
 public class Log4shellUpdate {
 
+  static boolean IS_WINDOWS = '\\' == File.separatorChar;
+
   private static final Pattern VERSION_PATTERN = Pattern.compile("-2\\.(\\d+)\\.\\d+\\.jar(\\.bak_log4shell)*");
   private static final Pattern DELETE_PATTERN = Pattern.compile("elasticsearch-sql-cli-\\d+\\.\\d+\\.\\d+\\.jar");
+  private static final Pattern LOG4J1_PATTERN = Pattern.compile("log4j-1.2.\\d+\\.jar");
   private static final Map<String, String> REPLACEMENTS;
   private static final List<String> VULNERABLE_CLASSES = Arrays.asList("org/apache/logging/log4j/core/lookup/JndiLookup.class");
   private static final Field ZIP_NAMES_FIELD;
+  static final String LOG4J2_WRAPPER_SETTINGS =
+      "\n" +
```

Comment:
We check for windows above. It might be that the files are with \r\n for windows formatted